NpmAddDependencies.js

```diff
@@ -14,11 +14,33 @@ class NpmAddDependencies {
     process.argv.forEach((val, index) => {
       if (val && index !== 0 && index !== 1) {
         switch (val) {
+          case '--prod':
+          case '--save-prod':
+          case '-P': {
+            this.target = 'dependencies';
+            break;
+          }
           case '--dev':
+          case '--save-dev':
+          case '--D': {
+            this.target = 'devDependencies';
+            break;
+          }
           case '--peer':
+          case '--save-peer': {
```

Comment:
There's no `--save-peer` and `--save-bundle` flags for `npm`, afaik. There IS `--save-peer` in yarn though.

Reply:
`--save-bundle` exists in `npm install`/`npm add` so I figured we could add. I agree on `--save-peer` since that doesn't exist in `npm`, though maybe worth having if it exists in `yarn`? I can remove.

Reply:
I think it's unnecessary since 1) we have the `default` switch section; 2) there's a line in the docs: 

> If no `target` argument passed, dependencies are written to `dependencies`.

Reply:
Well, we successfully removed the `--prod` flag, so we can use `-P` now for `--peer`?

Reply:
@arfeo Sure we can, but I'm not sure the ambiguity is great. `yarn` vs. `npm` users will not know what to expect.

Reply:
Well, the `--peer` flag seems to be orphaned without a shortcut... We can add `-P` to the docs, so npmers and yarners were able to get to know what to expect, can't we? =)

Reply:
So, will you add `-P` for `--peer`?

Reply:
@arfeo Done, added `-P` for `--peer` to the code and the README.